docs/guides/cloud/introduction.mdx

```diff
@@ -4,157 +4,153 @@ sidebar_position: 10
 sidebar_label: Introduction
 ---
 
-:::tip
+[Cypress Cloud](https://on.cypress.io/cloud) is our enterprise-ready companion
```

Comment:
I get an error every time I try to load this page
![Screenshot 2023-08-16 at 6 53 27 PM](https://github.com/cypress-io/cypress-documentation/assets/3980464/e836e987-64cf-488b-a7c0-3d4b96323474)

Reply:
actually I think its just the docusaurus skip to fallback content `#docusaurus_skipToContent_fallback` hash causing issues? Happens on a few other pages when I search content on the site

Reply:
I believe this is isolated to the local experience, these links are fine on prod and haven't changed. I'll look into it later tho.